goldmane/Makefile

```diff
@@ -48,6 +48,13 @@ protobuf proto/api.pb.go: proto/api.proto
 		$(CALICO_BUILD) sh -c 'protoc --proto_path=/code/proto --go_out=/code/proto --go-grpc_out=. --go_opt=paths=source_relative /code/proto/api.proto'
 	$(MAKE) fix-changed
 
+###############################################################################
+# Generate files
+###############################################################################
+GENERATED_FILES=protobuf/api.pb.go
+gen-files: $(GENERATED_FILES)
+	$(DOCKER_GO_BUILD) go run ./cmd/calico-felix-docgen --format=missing
```

Comment:
You're using felix's docgen tool?

Reply:
Sigh. I shouldn't just copy paste code without reading it, huh?